grakn-graql/src/test/java/ai/grakn/graql/internal/query/match/MatchTest.java

```diff
@@ -169,10 +170,9 @@
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertNotEquals;
 import static org.junit.Assert.assertThat;
-import static org.junit.Assert.assertTrue;
 
 @SuppressWarnings({"OptionalGetWithoutIsPresent", "unchecked"})
-public class MatchQueryTest {
+public class MatchTest {
```

Comment:
Maybe this should be `GetQueryTest` ? Based on all the actual test below?

Reply:
Maybe. The thing is, 99% of the behaviour we are testing is within the `match`. We only call `get` because stuff like the matchers and helper methods work only with a `GetQuery`. I'll try and make them more generic.

I should also try and split it into two tests: one small `GetQueryTest` and a big `MatchTest`.